lib/stm32/f4/rcc.c

```diff
@@ -785,8 +785,9 @@ uint32_t rcc_system_clock_source(void)
  * needed to establish a system clock.
  *
  * @param clock clock information structure.
+ * @param overdrive boolean to request OverDrive.
  */
-void rcc_clock_setup_pll(const struct rcc_clock_scale *clock)
+void rcc_clock_setup_pll_x(const struct rcc_clock_scale *clock, int overdrive)
```

Comment:
I think it would make more sense to call this function `rcc_clock_setup_pll_od` for OverDrive, it is more descriptive why there is a suffix to the function.